frontend/__test__/parseIsoStringFromDateTime.test.ts

```diff
@@ -0,0 +1,10 @@
+import { parseIsoStringFromDateTime } from '../src/utils/parseIsoStringFromDateTime';
+
+describe('parseIsoStringFromDateTime 유틸 테스트', () => {
+  it('날짜와 시간 정보를 받아 ISO 형태의 문자열을 반환한다.', () => {
+    const date = '2025-08-20';
+    const time = '20:57';
+    const result = '2025-08-20T11:57:00.000Z';
+    expect(parseIsoStringFromDateTime(date, time)).toBe(result);
```

Comment:
```suggestion
    const result = '2025-08-20T11:57:00.000Z';
    
    expect(parseIsoStringFromDateTime(date, time)).toBe(result);
```

다른 테스트 코드의 컨벤션에 맞게 한 줄 개행 부탁드려요!

Reply:
🫡 추가 완료했습니다! https://github.com/woowacourse-teams/2025-PhotoGather/pull/251/commits/250696788b5a578e436f1ca9aa14732bb7b0838c